Documentation/source/conf.py

```diff
@@ -236,7 +236,7 @@ def __getattr__(cls, name):
guments', 'NSWorkspaceLaunchConfigurationEnvironment', 'NSWorkspaceLaunchDefault', 'NSWorkspaceLaunchInhibitingBackgroundOnly', 'NSWorkspaceLaunchNewInstance', 'NSWorkspaceLaunchPreferringClassic', 'NSWorkspaceLaunchWithoutActivation', 'NSWorkspaceLaunchWithoutAddingToRecents', 'NSWorkspaceLinkOperation', 'NSWorkspaceMoveOperation', 'NSWorkspaceNotificationCenter', 'NSWorkspaceRecycleOperation', 'NSWorkspaceScreensDidSleepNotification', 'NSWorkspaceScreensDidWakeNotification', 'NSWorkspaceSessionDidBecomeActiveNotification', 'NSWorkspaceSessionDidResignActiveNotification', 'NSWorkspaceVolumeLocalizedNameKey', 'NSWorkspaceVolumeOldLocalizedNameKey', 'NSWorkspaceVolumeOldURLKey', 'NSWorkspaceVolumeURLKey', 'NSWorkspaceWillLaunchApplicationNotification', 'NSWorkspaceWillPowerOffNotification', 'NSWorkspaceWillSleepNotification', 'NSWorkspaceWillUnmountNotification', 'NSWrapCalendarComponents', 'NSWrapperCellView', 'NSWritingDirectionAttributeName', 'NSWritingDirectionLeftToRight', 'NSWritingDirectionNatural', 'NSWritingDirectionRightToLeft', 'NSXMLAttributeCDATAKind', 'NSXMLAttributeDeclaration', 'NSXMLAttributeDeclarationKind', 'NSXMLAttributeEntitiesKind', 'NSXMLAttributeEntityKind', 'NSXMLAttributeEnumerationKind', 'NSXMLAttributeIDKind', 'NSXMLAttributeIDRefKind', 'NSXMLAttributeIDRefsKind', 'NSXMLAttributeKind', 'NSXMLAttributeNMTokenKind', 'NSXMLAttributeNMTokensKind', 'NSXMLAttributeNotationKind', 'NSXMLChildren', 'NSXMLCommentKind', 'NSXMLContext', 'NSXMLDTD', 'NSXMLDTDKind', 'NSXMLDTDNode', 'NSXMLDocument', 'NSXMLDocumentHTMLKind', 'NSXMLDocumentIncludeContentTypeDeclaration', 'NSXMLDocumentKind', 'NSXMLDocumentMap', 'NSXMLDocumentMapNode', 'NSXMLDocumentTextKind', 'NSXMLDocumentTidyHTML', 'NSXMLDocumentTidyXML', 'NSXMLDocumentValidate', 'NSXMLDocumentXHTMLKind', 'NSXMLDocumentXInclude', 'NSXMLDocumentXMLKind', 'NSXMLElement', 'NSXMLElementDeclarationAnyKind', 'NSXMLElementDeclarationContent', 'NSXMLElementDeclarationElementKind', 'NSXMLElementDeclarationEmptyKind', 'NSXMLElementDeclarationKind', 'NSXMLElementDeclarationMixedKind', 'NSXMLElementDeclarationUndefinedKind', 'NSXMLElementKind', 'NSXMLEntityDeclarationKind', 'NSXMLEntityGeneralKind', 'NSXMLEntityParameterKind', 'NSXMLEntityParsedKind', 'NSXMLEntityPredefined', 'NSXMLEntityUnparsedKind', 'NSXMLFidelityElement', 'NSXMLFidelityNode', 'NSXMLInvalidKind', 'NSXMLNSArrayTransformerName', 'NSXMLNSDataTransformerName', 'NSXMLNSDateTransformerName', 'NSXMLNSNumberTransformerName', 'NSXMLNSURLTransformerName', 'NSXMLNamedFidelityNode', 'NSXMLNamedNode', 'NSXMLNamespaceKind', 'NSXMLNode', 'NSXMLNodeCompactEmptyElement', 'NSXMLNodeExpandEmptyElement', 'NSXMLNodeIsCDATA', 'NSXMLNodeOptionsNone', 'NSXMLNodePreserveAll', 'NSXMLNodePreserveAttributeOrder', 'NSXMLNodePreserveCDATA', 'NSXMLNodePreserveCharacterReferences', 'NSXMLNodePreserveDTD', 'NSXMLNodePreserveEmptyElements', 'NSXMLNodePreserveEntities', 'NSXMLNodePreserveNamespaceOrder', 'NSXMLNodePreservePrefixes', 'NSXMLNodePreserveQuotes', 
```

Comment:
Not sure what this file does, but I' pretty sure the change here is not correct. At best, the "new" constants should be added here (in alphabetical place), but the old ones should probably stay.

Reply:
this makes dummy classes and objects for the sphinx doc to work on other an OS